pkg/signup/signup_service_test.go

```diff
@@ -0,0 +1,107 @@
+package signup_test
+
+import (
+	"context"
```

Comment:
can we organize these imports?

i installed https://godoc.org/golang.org/x/tools/cmd/goimports - it works okay, does basic organization. also removes unused imports which is kind of nice...

Reply:
Neat tool, thanks for pointing it out :)  I've installed it into my GoLand environment so it now happens automatically when I save each file.